src/components/zenlytic-legend/ZenlyticLegend.js

```diff
@@ -58,16 +46,15 @@ const ZenlyticLegend = ({
       layout="vertical"
       align="right"
       verticalAlign={isServerSide ? 'top' : 'middle'}
-      iconSize={useStrokeColorShape ? 0 : iconType === 'line' ? 14 : 12}
       color={colors.gray[500]}
       wrapperStyle={{
```

Comment:
- [ ] Most important thing - make it not go out of the container, use overflow: auto (or hidden, but auto is better)